test/test_automated_install.py

```diff
@@ -488,3 +488,17 @@ def test_validate_ip_invalid_letters(Pihole):
     ''')
 
     assert output.rc == 1
+
+def test_os_check_passes(Pihole):
+    ''' Confirms OS meets the requirements '''
+    Pihole.run('''
+    source /opt/pihole/basic-install.sh
+    distro_check
+    install_dependent_packages ${INSTALLER_DEPS[@]}
+    ''')
+    detectOS = Pihole.run('''
+    source /opt/pihole/basic-install.sh
+    os_check
```

Comment:
```suggestion
    source /opt/pihole/basic-install.sh
    pretty_name = 'PRETTY_NAME="Debian GNU/Linux 10 (buster)"'
    with open('/etc/os-release','w') as f:
        f.write(pretty_name)
    os_check
```

Or something in that same vein.

Reply:
Might have to break out the python code to a line before `.run`, I don't remember if you can do python inside the `run` or not.

Reply:
The only thing was the file `*-release` file having different names in different distros...

Reply:
Okay, just change the name of the file you're writing to when you change the `pretty_name` string.